ledger/error/src/main/scala/com/daml/error/ErrorCode.scala

```diff
@@ -262,6 +262,8 @@ object ErrorCode {
   def formatContextAsString(contextMap: Map[String, String]): String = {
     contextMap
       .filter(_._2.nonEmpty)
+      .toSeq
+      .sortBy(_._1)
```

Comment:
Necessary to make the order of entry more predictable. Until now it was ~shortest entry first